.ci/pytorch/multigpu-test.sh

```diff
@@ -43,16 +43,15 @@ time python test/run_test.py --verbose -i distributed/_tensor/test_dtensor_compi
 time python test/run_test.py --verbose -i distributed/test_device_mesh
 
 # DTensor/TP tests
-time python test/run_test.py --verbose -i distributed/tensor/parallel/test_ddp_2d_parallel
-time python test/run_test.py --verbose -i distributed/tensor/parallel/test_fsdp_2d_parallel
 time python test/run_test.py --verbose -i distributed/tensor/parallel/test_tp_examples
 time python test/run_test.py --verbose -i distributed/tensor/parallel/test_tp_random_state
 
 # FSDP2 tests
 time python test/run_test.py --verbose -i distributed/_composable/fsdp/test_fully_shard_training -- -k test_2d_mlp_with_nd_mesh
 
-# Pipelining composability tests
-time python test/run_test.py --verbose -i distributed/pipelining/test_composability.py
+# 3D composability tests
+time python test/run_test.py --verbose -i distributed/_composable/test_composability/test_noncontinuous.py
+time python test/run_test.py --verbose -i distributed/_composable/test_composability/test_continuous.py
```

Comment:
could I know why they are new tests are called `test_noncontinuous` and `test_continuous` ?

Reply:
The `MultiProcContinousTest` at `test_continuous` relates to `MultiProcContinousTest`, thus name the test file as `test_continuous`.
The others tests relate the old style test. `test_noncontinuous` is not suitable, what do you think of `test_composability_fsdp_tp`